chart/Chart.yaml

```diff
@@ -15,7 +15,7 @@ keywords:
 - tracing
 - opentelemetry
 
-version: 13.1.0
```

Comment:
Since the `image` format change is not backwards compatible, I think we should bump the major version.